library.properties

```diff
@@ -1,5 +1,5 @@
 name=Nokia 5110 LCD library
-version=2.2.2
+version=2.3.1
```

Comment:
No reason to change it to `2.3.1` as `2.3.0` was never released. So it's still `2.3.0`. :+1:

Reply:
fixed

Reply:
Did you forget to push this? :scream_cat: